P1.txt

```diff
@@ -0,0 +1,12 @@
+val a: Integer
+read(a)
+
+if (a < 2) {
+	write(“false”)
+} else {
+	for (2 to a/2)

```

Comment:
The for statement does not fulfill the definition (forStatement  =  “for” “(” constant “to” constant “)” statement) as "a/2" is an expression, not a constant